pkg/controller/admission/validate_cluster.go

```diff
@@ -75,8 +75,12 @@ func (s *ClusterValidatingAdmissionWebhook) ValidateUpdate(old aerospikev1alpha1
 	}
 
 	// Jump version should not be allowed
-	newVersion := strings.Split(s.obj.Spec.Build, ":")[1]
-	oldVersion := strings.Split(old.Spec.Build, ":")[1]
+	newVersion := strings.Split(s.obj.Spec.Image, ":")[1]
+	oldVersion := ""
```

Comment:
Was it crashing? old image should always be there in case of update i think. Just curious!

Reply:
Oh yes. I had a few running clusters running before this rename :) They were getting stuck and not being deleted because the field name was "build"